canary/src/main/java/io/littlehorse/canary/config/ConfigLoader.java

```diff
@@ -0,0 +1,81 @@
+package io.littlehorse.canary.config;
+
+import io.smallrye.config.PropertiesConfigSource;
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import io.smallrye.config.common.utils.ConfigSourceUtil;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+import org.eclipse.microprofile.config.Config;
+
+public class ConfigLoader {
+
+    private ConfigLoader() {}
+
+    /**
+     * Loads properties in this order:
+     * 1. MicroProfile Config configuration file META-INF/microprofile-config.properties in the classpath
+     * 2. Environment variables
```

Comment:
Is `LITTLEHORSE_CANARY_FOO` translated to `littlehorse.canary.foo`? If so, we should migrate the LH Server to use this as well. That is really cool.